packages/x-data-grid-pro/src/tests/editComponents.DataGridPro.test.tsx

```diff
@@ -598,7 +598,7 @@ describe('<DataGridPro /> - Edit components', () => {
       const cell = getCell(0, 0);
       await user.dblClick(cell);
       await user.click(screen.queryAllByRole('option')[1]);
-      await waitFor(() => expect(screen.queryByRole('listbox')).to.equal(null));
+      expect(screen.queryByRole('listbox')).to.equal(null);
       await act(() => {
```

Comment:
😬 
```suggestion
      await act(async () => {
```
Fine for me to do a separate pass of these, but we should either remove the `await` or add the `async`.